src/molecules/formfields/CreditCardField/Readme.md

```diff
@@ -1,5 +1,27 @@
 CreditCardField Example:
 
 ```example
-    <CreditCardField />
+    <CreditCardField>
+      <CreditCardField.Top>
+        <TextField
+          placeholder='Credit Card Number'
+          noBaseStyle
+        />
+      </CreditCardField.Top>
+
+      <CreditCardField.Bottom>
+        <TextField
+          placeholder='Expiration Date'
+          input={{
+            pattern: '[0-9]{13,16}'
```

Comment:
@jmcolella it's a html input attribute - you can pass a regex to control the input. @patrickkim added this in originally though & tbh I couldn't figure out if it was working correctly because it didn't seem to be limiting the input in any way